sql-statements/sql-statement-select.md

```diff
@@ -47,10 +63,19 @@ SelectLockOpt ::=
 TableList ::=
     TableName ( ',' TableName )*
 
+WhereClause ::=
+    "WHERE" Expression ( ("AND" | "OR") Expression)*
```

Comment:
```suggestion
    "WHERE" Expression
```

`expr "AND" expr` itself is a valid expression, no need to give the `"AND" | "OR"` any special treatment